### PR TITLE
Added Error Handler Component

### PR DESCRIPTION
**Pull request by prathambatra**
Signed-off-by: prathambatra <batrapratham999@gmail.com>

### CODE REVIEW

**Comment by 0xnmn** [comment]
Add screenshots

File: app/components/ErrorHandle/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const ErrorHandle = () => (
+  <div className="err-message">
+    <h2>
+      There was some error.{' '}
+      <a href="/" class="orange">
+        Back to Home
+      </a>
+    </h2>
+    <img
+      src="https://minio.codingblocks.com/amoeba/404-image-min.png"
```
Comment (0xnmn) [inline app/components/ErrorHandle/index.js]: Copy the image in this project.

**Review by 0xnmn — CHANGES_REQUESTED** [review]

File: app/components/ErrorHandle/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './style.scss';
+
+const ErrorHandle = () => (
+  <div className="err-message">
+    <h2>
+      There was some error.{' '}
+      <a href="/" class="orange">
```
Comment (0xnmn) [inline app/components/ErrorHandle/index.js]: not required.

Comment (prathambatra) [reply app/components/ErrorHandle/index.js]: CSS not required?

File: app/components/ErrorHandle/index.js (hunk 1 above)
Comment (prathambatra) [reply app/components/ErrorHandle/index.js]: okay

File: app/components/ErrorHandle/index.js (hunk 2 above)
Comment (0xnmn) [reply app/components/ErrorHandle/index.js]: even the link and text

**Comment by prathambatra** [comment]
@hereisnaman please see this new PR #50

**Pull request closed without merging**